saleor/app/manifest_validations.py

```diff
@@ -178,31 +182,93 @@ def clean_extensions(manifest_data, app_permissions, errors):
         _clean_extension_permissions(extension, app_permissions, errors)
 
 
+def clean_webhooks(manifest_data, errors):
+    webhooks = manifest_data.get("webhooks", [])
+
+    async_types = {
+        str_to_enum(e_type[0]): e_type[0] for e_type in WebhookEventAsyncType.CHOICES
+    }
+    sync_types = {
+        str_to_enum(e_type[0]): e_type[0] for e_type in WebhookEventSyncType.CHOICES
+    }
+
+    target_url_validator = AppURLValidator(
+        schemes=["http", "https", "awssqs", "gcpubsub"]
+    )
+
+    for webhook in webhooks:
+        if not validate_subscription_query(webhook["query"]):
+            errors["webhooks"].append(
+                ValidationError(
+                    "Subscription query is not valid.",
```

Comment:
Would be good to provide here more context. Currently, we know that the subscription query is invalid but we don't know which webhook declaration has an issue.

Reply:
I agree with that, but at the same time I wanted to make it coherent with how we validate extensions - we also do not provide mean to identify, which one is causing validation error (see: https://github.com/saleor/saleor/blob/main/saleor/app/manifest_validations.py#L160)

* should I implement that in this pull request? should I also change the way `extensions` errors are defined?
* is changing shape of `errors` object considered breaking change?